content/applications/sales/crm/track_leads/lead_distribution_report.rst

```diff
@@ -0,0 +1,218 @@
+field and select :guilabel:`contains` from the drop-down menu. Selecting this operator will filter
+for any records that contain the words in the rightmost field.
+
+In this third field, enter the name of the desired sales teams. For example, setting `us direct sf
+northam` will include only sales teams assigned to the U.S., direct, San Francisco, and North
+America.
+
+.. image:: lead_distribution_report/sales-team-location.png
+   :align: center
+   :alt: Use Sales Team to filter the location the lead is associated with.
+
+Add phone number
+~~~~~~~~~~~~~~~~
+
+Click :guilabel:`New rule` to add a new rule to the filter. Set the first field to
+:guilabel:`Phone`. Then, click the next field and select :guilabel:`is set` from the drop-down menu.
+Selecting this operator will filter only for records that have a phone number associated with the
+lead.
+
+.. image:: lead_distribution_report/phone-set.png
+   :align: center
+   :alt: Report only leads with an associated phone number with the Phone field.
+
+Add active status
+~~~~~~~~~~~~~~~~~
+
+Click the :guilabel:`Branch` icon to the right of :guilabel:`Phone is set` to add a new rule that
+branches from the rules above.
+
+Two horizontal sets of fields will appear below a line showing
+:guilabel:`all of:` with a :icon:`fa-caret-down` down arrow. Click the down arrow, then select
+:guilabel:`any` from the list.
+
+Set the first field to :guilabel:`Active`. Then, click the rightmost field and select
+:guilabel:`is set` from the drop-down menu.
+
+.. image:: lead_distribution_report/add-branch.png
+   :align: center
+   :alt: Click the second button to the right of phone is set labeled add branch.
+
+Next, click the :icon:`fa-plus` :guilabel:`New Rule` button next to :guilabel:`Active is set` to
+create a new set of fields. Set the first field to :guilabel:`Active`. Then, click the next field
+and select :guilabel:`is not set` from the drop-down menu. This rule will add the activity status of
+the lead to the report.
+
+.. image:: lead_distribution_report/active-set.png
+   :align: center
+   :alt: Use Active to include active status in the report.
+
+Add referred by
+~~~~~~~~~~~~~~~
+
+Click the :guilabel:`Branch` icon to the right of :guilabel:`any of:` field to add a new set of
+rules that branches from the rules above and another horizontal set of fields will appear below.
+Change :guilabel:`all of:` to :guilabel:`any of:`.
+
+Next, set the first field to :guilabel:`Referred By`. In the field to the right, select
+:guilabel:`contains` from the drop-down menu. In the last field, type `appointment`. This will add
+any leads that were referred by an appointment.
+
+.. image:: lead_distribution_report/referred-by.png
+   :align: center
+   :alt: Add referred by as a branch to filter appointments.
+
+Add source
+~~~~~~~~~~
+
+Click the :icon:`fa-plus` :guilabel:`New Rule` button next to :guilabel:`Referred by appointment`.
```

Comment:
what about the second line of the newly-created 'branch' that's still on the default 'country' at this point? Does that need to be mentioned here?